grpcmock-spring-boot/src/main/java/org/grpcmock/springboot/GrpcMockApplicationListener.java

```diff
@@ -36,6 +38,14 @@ private void registerPort(ConfigurableEnvironment environment) {
       environment.getPropertySources().addFirst(properties);
       properties.getSource().put("grpcmock.server.port", availablePort);
       properties.getSource().put("grpcmock.server.port-dynamic", true);
+    } else if (httpPort.equals(-1)) {
```

Comment:
Maybe it's better to use some boolean flag, rather than some hardcoded port value?

Reply:
I could create a boolean flag, but I wanted to be more consistent and don't create redundant flags. I mean we already have "magic" port value of zero for random port, so I thought -1 will be ok.
Another option is to check for server name and if it's set use in-process server. But in this case we lose the ability to generate server name.
So we can use:
* port -1 as implemented
* additional boolean flag like `useInProcessServer`
* check that server name property is set and use in-process server instead of TCP-one.
I can do any, which do you prefer?

Reply:
Yeah, but port value of `0` is generally accepted constant, used in spring and other libraries. I'd still say that boolean flag is better in this regard, though the 3rd approach could work as well (and for random name it could be set in the annotation itself `@AutoConfigureGrpcMock(name = InProcessServerBuilder.generateName())`)
Also if fixed name is used we should mark the context as dirty in `GrpcMockTestExecutionListener`, same as with a fixed port value.

Reply:
Made it with boolean flag and marked context as dirty as well.
One thing not very pleasant is `grpcmock.server.port-dynamic` property which is now used for in process server name too. I could rename it but it would break backwards compatibility. I could also add new one, mark old one with `@Deprecated` and set both of them.
Would you like me to do this or we'll leave it as is?

Reply:
I think it's fine as is. Thanks